src/apify_client/clients/resource_clients/build.py

```diff
@@ -43,16 +43,16 @@ def abort(self) -> dict:
         """
         return self._abort()
 
-    def get_open_api_specification(self) -> dict | None:
-        """Return OpenAPI specification of the Actor's build.
+    def get_open_api_definition(self) -> dict | None:
```

Comment:
isn't it better this way? the endpoint as well as the docs use `openapi` instead of `open-api`.

```suggestion
    def get_openapi_definition(self) -> dict | None:
```

Reply:
Hm, if we rename it in python, it will also make sense to rename it in js client:

`getOpenApiDefinition()` to `getOpenapiDefinition`

But for me `getOpenApiDefinition()` looks cleaner, what do you think?

Reply:
as mentioned in the edit, it indeed feels weird in the JS version, so lets leave it the way you have it